### DISCUSSION TITLE
Copilot stopped working on Intellij EAP versions

### DISCUSSION THREAD

**Discussion by NickAcPT** [community/community#14249] [Copilot Conversations] [answered]
I am using IntelliJ IDEA 2022.1 Beta, build 221.5080.93 and Copilot (version 1.1.15.1127-agent) just refuses to work. The copilot agent launches fine, but appears to just sit idly and refusing to work ![](https://i.imgur.com/3QAH76t.png)

The copilot tool panel only displays the following:
![](https://i.imgur.com/dq8wukD.png)

Pressing refresh does nothing and I have tried having a look at the logs, but... there's no logs.. Couldn't find a single log file anywhere so I am unable to help debug this issue any further.

I have also tried re-logging in on the extension (Tools -> GitHub Copilot) and even reinstalling the extension.

**Comment by jansorg** [comment, marked as answer]
Thank you for you report! We're looking into it now, it seems to be a bug.

**Reply by NickAcPT** [reply]
Thanks a lot! If there's anything I can help with debugging this issue from my side, please let me know. 👍

**Reply by bmuskalla** [reply]
Thanks for the offer @NickAcPT - we've been able to reproduce the problem and found the culprit. Please bear with us for a bit so we can publish the bug fix onto the nightly channel. Thanks for reporting this! We'll report here once the new version is available

**Reply by bruno-oliveira** [reply]
Awesome work guys! :D By the way, Copilot is absolutely amazing for aiding with work and suggestions for boilerplate and writing fully-fledged complete unit tests, its amazing!
